setup.py

```diff
@@ -27,6 +27,7 @@
     readme = f.read()
 
 kerberos_require = ["requests_kerberos"]
+gssapi_require = ["requests_kerberos"]
```

Comment:
this is same as `kerberos_require`. Why is this needed separately?

Reply:
spot on, this is a typo. fixed